### PR TITLE
Pln1.1

### PR DESCRIPTION
**Pull request by Wu-Ca**
Updated PLN 1.1 content.

### CODE REVIEW

File: content/en/curriculum/pln/intro-to-pl/index.md [hunk 1]
```diff
@@ -0,0 +1,203 @@
...
+
+Protocol Labs enables a network of projects, tools, startups, research, communities (and more) to spring up and evolve around new protocols and web3 technologies. As this ecosystem grows, these projects invest in the network itself, and the network grows along with it, which in turn benefits and incentivizes this ongoing evolution.
+
+Protocol Labs, at its core, is an organization (also referred to internally as Starfleet) that supports open-source software projects, engineering research in multiple fields related to computation, communication, and knowledge, as well as other services. PL also supports the network of organizations that builds upon and expands the capabilities of its core technologies.
+
+Protocol Labs is building the next generation of the internet. We are an open-source Network that spans the research-to-development pipeline to create new protocols, tools, and services to improve the internet radically. Our products serve thousands of organizations and millions of people.
```
Comment (walkerlj0) [inline content/en/curriculum/pln/intro-to-pl/index.md:35]:
```suggestion
Protocol Labs is building the next generation of the internet. We are an open-source network that spans the research-to-development pipeline to create new protocols, tools, and services to improve the internet radically. Our products serve thousands of organizations and millions of people.
```

File: content/en/curriculum/pln/intro-to-pl/index.md [hunk 2]
```diff
@@ -0,0 +1,203 @@
...
+
+Many teams in the PL Network are fully distributed and follow a standard set of open-source practices to grow and flourish together. This distributed, open-source philosophy is at the core of how PL operates.
```
Comment (walkerlj0) [inline content/en/curriculum/pln/intro-to-pl/index.md:37]: Add a link here to the network directory at: https://www.plnetwork.io/directory/teams

**Comment by walkerlj0** [comment]
@Wu-Ca I made a couple edits (review not complete yet)

**Review by walkerlj0 — CHANGES_REQUESTED** [review]
Have some change requests

File: content/en/curriculum/pln/intro-to-pl/index.md [hunk 3]
```diff
@@ -0,0 +1,203 @@
...
+
+Protocol Labs, at its core, is an organization (also referred to internally as Starfleet) that supports open-source software projects, engineering research in multiple fields related to computation, communication, and knowledge, as well as other services. PL also supports the network of organizations that builds upon and expands the capabilities of its core technologies.
+
+Protocol Labs is building the next generation of the internet. We are an open-source network that spans the research-to-development pipeline to create new protocols, tools, and services to improve the internet radically. Our products serve thousands of organizations and millions of people.
+
+Many teams in the PL Network are fully distributed and follow a standard set of open-source practices to grow and flourish together. This distributed, open-source philosophy is at the core of how PL operates.
```
Comment (Wu-Ca) [inline content/en/curriculum/pln/intro-to-pl/index.md:37]:
```suggestion
Many [teams in the PL Network](https://www.plnetwork.io/directory/teams)  are fully distributed and follow a standard set of open-source practices to grow and flourish together. This distributed, open-source philosophy is at the core of how PL operates.
```

**Comment by walkerlj0** [comment]
Will re-do this PR

**Pull request closed without merging**